examples/relay-treasurehunt/build/babelGraphQLPlugin.js

```diff
@@ -1,9 +1,4 @@
-var fs = require('fs');
-var getBabelRelayPlugin = require('babel-relay-plugin');
-var path = require('path');
+var getBabelGraphQLPlugin = require('babel-relay-plugin');
```

Comment:
Good point. I'll change it to getBabelRelayPlugin since that's what it's actually called.